core/src/main/java/org/infinispan/commands/write/ApplyDeltaCommand.java

```diff
@@ -55,14 +40,7 @@ public Delta getDelta(){
     */
    @Override
    public Object perform(InvocationContext ctx) throws Throwable {
-      CacheEntry contextEntry = ctx.lookupEntry(key);
-      if (contextEntry instanceof DeltaAwareCacheEntry) {
-         DeltaAwareCacheEntry deltaAwareCacheEntry = (DeltaAwareCacheEntry) contextEntry;
-         deltaAwareCacheEntry.appendDelta(delta);
-      } else {
-         throw new IllegalStateException();
-      }
-      return null;
+      throw new UnsupportedOperationException();
```

Comment:
You know that the publicity of interceptors is problematic. I left this as someone might consider it public, and removing the method from `Visitor` could break something.